Rubberduck.Deployment/Rubberduck.Deployment.csproj

```diff
@@ -58,15 +56,7 @@
       <Output TaskParameter="Value" PropertyName="TargetAssembly" />
     </CreateProperty>
 
-    <RubberduckPostBuildTask 
-        Config="$(ConfigurationName)" 
-        NetToolsDir="$(SdkPath)bin\NETFX 4.6.1 Tools\" 
-        WixToolsDir="$(ProjectDir)WixToolset\" 
-        SourceDir="$(TargetDir)" 
-        TargetDir="$(TargetDir)" 
-        ProjectDir="$(ProjectDir)" 
-        IncludeDir="$(ProjectDir)InnoSetup\Includes\" 
-        FilesToExtract="Rubberduck.dll|Rubberduck.API.dll" />
+    <RubberduckPostBuildTask Config="$(ConfigurationName)" NetToolsDir="$(SdkPath)bin\NETFX 4.6.1 Tools\" WixToolsDir="$(ProjectDir)WixToolset\" SourceDir="$(TargetDir)" TargetDir="$(TargetDir)" ProjectDir="$(ProjectDir)" IncludeDir="$(ProjectDir)InnoSetup\Includes\" FilesToExtract="Rubberduck.dll|Rubberduck.API.dll" />
```

Comment:
I'd prefer this be reverted. Due to the number of parameters, it's unwieldy to have to read to right rather than top to bottom.

Reply:
Didn't realize this had occurred. #TIL